product_abc_classification/readme/USAGE.md

```diff
@@ -0,0 +1,13 @@
+To use this module, you need to:
+
+#\. Go to Sales or Inventory menu, then to Configuration/Products/ABC
+Classification Profile and create a profile with levels, knowing that
+the sum of all levels in the profile should sum 100 and all the levels
+should be different.
+
+#\. Later you should go to product categories or product variants, and
```

Comment:
I found no place to assign the profile on Product Category form view
![image](https://github.com/user-attachments/assets/6475678e-5a7c-40df-9b81-afdd1f2a7caf)